nautobot/dcim/tests/integration/test_controller.py

```diff
@@ -19,6 +20,7 @@ def test_controller_bulk_edit(self):
 
         # Create LocationType
         self.click_navbar_entry("Organization", "Location Types")
+        WebDriverWait(self.browser, 10).until(lambda b: b.url.endswith(reverse("dcim:locationtype_list")))
```

Comment:
Should we enhance the `click_navbar_entry` method to include an appropriate wait instead? Not fully sure I understand why this is needed in this set of tests but not others.